### DISCUSSION TITLE
Tables Not Rendering In GitHub Pages

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23945] [Other Feature Feedback, Questions, & Ideas]
<p>In Issue <a href="https://github.com/krishdevdb/reseter.css/issues/25" rel="noopener nofollow ugc">#25</a> on my repo <a href="https://github.com/krishdevdb/reseter.css" rel="noopener nofollow ugc">reseter.css</a>. I got to know that github pages doesn’t render markdown tables properly. I use the same readme as my repo it renders perfectly on <a href="http://github.com" rel="noopener nofollow ugc">github.com</a> but inappropriately on <a href="http://github.io" rel="noopener nofollow ugc">github.io</a>. I recreated the branch from master no changes. Then I went further and changed the github pages branch to master. Here’s the URL</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <a href="https://krishdevdb.github.io/reseter.css/" target="_blank" rel="noopener nofollow ugc">reseter.css</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://krishdevdb.github.io/reseter.css/" target="_blank" rel="noopener nofollow ugc">reseter.css</a></h3>

<p>Reseter.css - A Futuristic Alternative To Normalize.css And CSS Resets</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>And This is the github url of the readme</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/krishdevdb/reseter.css" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/360;"><img src="https://user-images.githubusercontent.com/76587897/181080346-a3a874b5-3bfb-4eaf-a114-558c75a6871d.png" class="thumbnail" width="690" height="360" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/d/5d2dfb8a52cc8f91bd2fe51e846d7624e4db35b2_2_690x360.png, https://aws1.discourse-cdn.com/github/optimized/3X/5/d/5d2dfb8a52cc8f91bd2fe51e846d7624e4db35b2_2_1035x540.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/5/d/5d2dfb8a52cc8f91bd2fe51e846d7624e4db35b2.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/d/5d2dfb8a52cc8f91bd2fe51e846d7624e4db35b2_2_10x10.png"></div>

<h3><a href="https://github.com/krishdevdb/reseter.css" target="_blank" rel="noopener nofollow ugc">krishdevdb/reseter.css</a></h3>

  <p><span class="label1">Reseter.css - A Futuristic Alternative To Normalize.css And CSS Resets</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>More Over Here’s The Issue For More Info</p>
<aside class="onebox githubissue">
  <header class="source">
      <a href="https://github.com/krishdevdb/reseter.css/issues/25" target="_blank" rel="noopener nofollow ugc">github.com/krishdevdb/reseter.css</a>
  </header>
  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Issue">
	  <svg width="60" height="60" class="github-icon" viewbox="0 0 14 16" aria-hidden="true"><path d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/krishdevdb/reseter.css/issues/25" target="_blank" rel="noopener nofollow ugc">Docs:  Reseter.css Web Site Display Bug </a>
    </h4>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2021-04-13" data-time="03:32:24" data-timezone="UTC">03:32AM - 13 Apr 21 UTC</span>
      </div>

      <div class="user">
        <a href="https://github.com/yamanoku" target="_blank" rel="noopener nofollow ugc">
          <img alt="yamanoku" src="https://user-images.githubusercontent.com/76587897/181080347-9bdc25ec-d8bc-457e-82c8-27e3c5ec0750.png" class="onebox-avatar-inline" width="20" height="20">
          yamanoku
        </a>
      </div>
    </div>
  </div>
</div>

<div class="github-row">
  <p class="github-content">Describe the bug
Under the Tweet and GitHub star buttons display bug
To Reproduce
Go to https://krishdevdb.github.io/reseter.css/
See bug
Expected behavior
Not display...</p>
</div>

<div class="labels">
    <span style="display:inline-block;margin-top:2px;background-color: #B8B8B8;padding: 2px;border-radius: 4px;color: #fff;margin-left: 3px;">documentation</span>
</div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>A Screen Shot As Well</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/f/0f6673284242d258c4e2a174b3aea7161ec232e6.png" data-download-href="/uploads/short-url/2ceEUjBTdNGaV534EyHX4ox0BnM.png?dl=1" title="" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/76587897/181080348-d8733140-5ff5-41e8-820c-0188b3d4f34b.png" alt="" data-base62-sha1="2ceEUjBTdNGaV534EyHX4ox0BnM" width="690" height="462" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0f6673284242d258c4e2a174b3aea7161ec232e6_2_690x462.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0f6673284242d258c4e2a174b3aea7161ec232e6_2_1035x693.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0f6673284242d258c4e2a174b3aea7161ec232e6_2_1380x924.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0f6673284242d258c4e2a174b3aea7161ec232e6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename"></span><span class="informations">2256×1512 390 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I think this must be enough to take action on</p>

**Comment by otter-computer** [comment, marked as answer]
<p>This is probably due to the slight differences in the default markdown renderer between Pages and <a href="http://GitHub.com">GitHub.com</a>. Pages uses kramdown by default whereas GitHub uses GitHub Flavoured Markdown. You can change this by <a href="https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/setting-a-markdown-processor-for-your-github-pages-site-using-jekyll">changing your markdown renderer</a> to GFM by setting <code>markdown: GFM</code> inside your <code>_config.yml</code> file.</p>
